job_creator.php

```diff
@@ -248,16 +237,15 @@ private function isAllowedPhpVersion(string $phpVersion)
      * Get the branch name from the installer version and left only the minor version
      * e.g. 4.10.x-dev -> 4.10
      */
-    private function getBranchName(): string
+    private function getInstallerBranch(): string
```

Comment:
Renamed to be clearer what this actually is - it's not the branch name in the repo, it's the name of the branch of installer that will be pulled in via composer.